packages/react-scripts/template/README.md

```diff
@@ -468,6 +468,18 @@ Then we can change `start` and `build` scripts to include the CSS preprocessor c
 
 Now running `npm start` and `npm run build` also builds Sass files. Note that `node-sass` seems to have an [issue recognizing newly created files on some systems](https://github.com/sass/node-sass/issues/1891) so you might need to restart the watcher when you create a file until it’s resolved.
 
+**Performance Note**
+
+`node-sass --watch` has been reported to have *performance issues*, especially when running inside a virtual machine such as docker. If you are experiencing high CPU usage with node-sass you can alternatively try [node-sass-chokidar](https://www.npmjs.com/package/node-sass-chokidar) which uses a different file-watcher. Usage remains the same, simply replace `node-sass` with `node-sass-chokidar`.
```

Comment:
I know it may seem redundant, but can you please show commands to uninstall and reinstall the new package?